Kernel/Net/Ethernet.h

```diff
@@ -8,8 +8,17 @@
 
 #include <AK/Endian.h>
 #include <AK/MACAddress.h>
+#include <AK/Types.h>
 
-#pragma GCC diagnostic ignored "-Warray-bounds"
```

Comment:
Removing this diagnostic will probably break at least the GCC build. If it is actually not required anymore, please make a separate commit removing it, since in general we shouldn't ignore diagnostics unless the compiler is very clearly wrong about something (and that usually doesn't happen unless we're doing whacky bleeding-edge-features things).